### ISSUE TITLE
Dolphin installation failed

### ISSUE THREAD

**Issue by stephanemagnenat** [haraldF/homebrew-kf5#35]
The installation of Dolphin head (April 8, 2015) failed. The error is the following:
```
==> Cloning git://anongit.kde.org/kde-baseapps.git
Updating /Library/Caches/Homebrew/kf5-dolphin--git
==> Checking out branch frameworks
==> Patching
patching file CMakeLists.txt
Hunk #1 FAILED at 41.
1 out of 1 hunk FAILED -- saving rejects to file CMakeLists.txt.rej
can't find file to patch at input line 27
Perhaps you used the wrong -p or --strip option?
The text leading up to this was:
--------------------------
|diff --git a/dolphin/CMakeLists.txt b/dolphin/CMakeLists.txt
|index 43e3807..c3c448f 100644
|--- a/dolphin/CMakeLists.txt
|+++ b/dolphin/CMakeLists.txt
--------------------------
No file to patch.  Skipping patch.
patch unexpectedly ends in middle of line
1 out of 1 hunk ignored
```

**Comment by nbigaouette** [comment]
I'm taking a look at this and it it seems dolphin was moved out of that repo? I think these are "old" KDE4 applications.

There is a Dolphin repo here: git://anongit.kde.org/dolphin.git (see https://projects.kde.org/projects/kde/applications/dolphin/repository ).

In any case, the patches applied in the formula are out of date.

**Comment by nbigaouette** [comment]
Note that even if PR #38 fixes the compilation, dolphin does not work properly:

![dolphin](https://cloud.githubusercontent.com/assets/112988/7208721/c5682fe4-e510-11e4-895f-f08eeb98c3a9.png)

There is no icons, the menu is "No text" for every menu name, etc. At least it doesn't crash.

I suspect it's the same issue as in #19.

**Comment by krf** [comment]
I've merged #38. The other issue you have looks related to #19 indeed. Closing.
